Config.cmake/test_fmi3.cmake

```diff
@@ -117,6 +112,8 @@ add_catch2_test(fmi3_import_model_structure_test         FMI3)
 add_catch2_test(fmi3_variability_causality_initial_test  FMI3)
 add_catch2_test(fmi3_import_bouncingball_xml_test        FMI3)
 add_catch2_test(fmi3_import_interface_types_test         FMI3)
+add_catch2_test(fmi3_import_unit_test                    FMI3)
```

Comment:
Name might not be optimal, open to better suggestions.

Reply:
fmi3_import_unit_definitions_test ?

Reply:
Fixed.